compliance_checker/ioos.py

```diff
@@ -6,11 +6,14 @@
                                      attr_check)
 from owslib.namespaces import Namespaces
 from lxml.etree import XPath
+from cf_units import Unit
 from compliance_checker.acdd import ACDD1_3Check
+from compliance_checker import cfutil
```

Comment:
Why not just import `get_z_variables` in the line below with the other methods?

Reply:
This isn't an exceptional use case of relative vs. qualified imports.  I'm probably going to keep it the same since it largely comes down to semantics.

Reply:
Oh, I see.  There's already some relative imports from cfutil.  I'll change it over.